client/blocks/upgrade-nudge-expanded/index.jsx

```diff
@@ -149,6 +149,7 @@ UpgradeNudgeExpanded.propTypes = {
 	eventName: PropTypes.string,
 	event: PropTypes.string,
 	siteSlug: PropTypes.string,
+	shouldDisplay: PropTypes.bool,
```

Comment:
`forceDisplay` would be a better name. If `forceDisplay === false`, the component can still be displayed if `currentPlan` is provided. With `shouldDisplay === false`, the same sentence sounds a bit confusing.